src/catalog/database_catalog.cpp

```diff
@@ -1869,432 +1848,294 @@ void DatabaseCatalog::BootstrapProcs(const common::ManagedPointer<transaction::T
+
+  BootstrapProcContext(txn, postgres::POW_PRO_OID, "pow", type::TypeId::DECIMAL, {type::TypeId::DECIMAL},
+                       execution::ast::Builtin::Pow, false);
+
+  BootstrapProcContext(txn, postgres::SPLIT_PART_PRO_OID, "split_part", type::TypeId::VARCHAR,
+                       {type::TypeId::VARCHAR, type::TypeId::VARCHAR, type::TypeId::INTEGER},
+                       execution::ast::Builtin::SplitPart, true);
+
+  BootstrapProcContext(txn, postgres::CHR_PRO_OID, "chr", type::TypeId::VARCHAR, {type::TypeId::INTEGER},
+                       execution::ast::Builtin::Chr, true);
+
+  BootstrapProcContext(txn, postgres::CHARLENGTH_PRO_OID, "char_length", type::TypeId::INTEGER, {type::TypeId::VARCHAR},
+                       execution::ast::Builtin::CharLength, true);
+
+  BootstrapProcContext(txn, postgres::POSITION_PRO_OID, "position", type::TypeId::INTEGER,
+                       {type::TypeId::VARCHAR, type::TypeId::VARCHAR}, execution::ast::Builtin::Position, true);
+
+  BootstrapProcContext(txn, postgres::LENGTH_PRO_OID, "length", type::TypeId::INTEGER, {type::TypeId::VARCHAR},
+                       execution::ast::Builtin::Length, true);
+
+  BootstrapProcContext(txn, postgres::UPPER_PRO_OID, "upper", type::TypeId::VARCHAR, {type::TypeId::VARCHAR},
+                       execution::ast::Builtin::Upper, true);
+
+  BootstrapProcContext(txn, postgres::VERSION_PRO_OID, "version", type::TypeId::VARCHAR, {},
+                       execution::ast::Builtin::Version, true);
+
+  BootstrapProcContext(txn, postgres::STARTSWITH_PRO_OID, "starts_with", type::TypeId::BOOLEAN,
+                       {type::TypeId::VARCHAR, type::TypeId::VARCHAR}, execution::ast::Builtin::StartsWith, true);
+
+  BootstrapProcContext(txn, postgres::SUBSTR_PRO_OID, "substr", type::TypeId::VARCHAR,
+                       {type::TypeId::VARCHAR, type::TypeId::INTEGER, type::TypeId::INTEGER},
+                       execution::ast::Builtin::Substring, true);
+
+  BootstrapProcContext(txn, postgres::REVERSE_PRO_OID, "reverse", type::TypeId::VARCHAR, {type::TypeId::VARCHAR},
+                       execution::ast::Builtin::Reverse, true);
+
+  BootstrapProcContext(txn, postgres::LEFT_PRO_OID, "left", type::TypeId::VARCHAR,
+                       {type::TypeId::VARCHAR, type::TypeId::INTEGER}, execution::ast::Builtin::Left, true);
+
+  BootstrapProcContext(txn, postgres::RIGHT_PRO_OID, "right", type::TypeId::VARCHAR,
+                       {type::TypeId::VARCHAR, type::TypeId::INTEGER}, execution::ast::Builtin::Right, true);
+
+  BootstrapProcContext(txn, postgres::REPEAT_PRO_OID, "repeat", type::TypeId::VARCHAR,
+                       {type::TypeId::VARCHAR, type::TypeId::INTEGER}, execution::ast::Builtin::Repeat, true);
+
+  BootstrapProcContext(txn, postgres::TRIM_PRO_OID, "btrim", type::TypeId::VARCHAR,
+                       {type::TypeId::VARCHAR, type::TypeId::VARCHAR}, execution::ast::Builtin::Trim, true);
```

Comment:
I'm just noticing this now, but I feel like this is a mistake (not introduced by this PR). This has the same signature as TRIM2 below, and looking at semantic checking/bytecode generation it looks like this should only have 1 argument. From testing though it seems to work with one arg, not really sure how.

Reply:
I thought that looked off. If you don't mind, could you push a fix to the branch when you get a chance? You should have access since you're a reviewer.

Reply:
yep, I'll update this and Concat. The rest looks good to me.